src/ansible_navigator/__init__.py

```diff
@@ -1,2 +1,14 @@
-"""The ansible-navigator application."""
-from ._version import __version__  # noqa: F401
+"""The ansible-navigator application.
+
+This is a transitional configuration during the migration
+from the hardcoded __version__ in _version to
```

Comment:
this line contains non-wrapped identifiers

Reply:
It's only going to be there for a couple hours.